File: Programmers/Inho/Level01/최소직사각형.cs
using System;

public class Solution
{
    public int solution(int[,] sizes)
    {
        int answer = 0;
        int big = 0;
        int small = 0;
        for (int i = 0; i < sizes.Length / 2; i++)
        {
            big = Math.Max(big, Math.Max(sizes[i, 0], sizes[i, 1]));
            small = Math.Max(small, Math.Min(sizes[i, 0], sizes[i, 1]));

        }
        return big * small;
    }
}

File: Programmers/Inho/C++/Level01/최소직사각형.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(vector<vector<int>> sizes) {
    int answer = 0;
    int small, big;
    int rSmall = 0;
    int rBig = 0;
    for(int i = 0;i< sizes.size();i++){
        small = min(sizes[i][0], sizes[i][1]);
        big = max(sizes[i][0], sizes[i][1]);
        rSmall = max(rSmall, small);
        rBig = max(rBig, big);
        
    }
    answer = rSmall * rBig;
    
    return answer;
}